sc/social/like/subscribers.py

```diff
@@ -97,9 +98,7 @@ def assign_canonical_url(obj, event):
 
     # we can't assign a canonical URL without a canonical domain
     if canonical_domain:
-        # FIXME: we're currently ignoring the Plone site id
-        #        https://github.com/collective/sc.social.like/issues/119
-        path = '/'.join(obj.getPhysicalPath()[2:])
+        path = get_path_to_virtual_path(event.request, obj)
```

Comment:
While reviewing #118, I had a bad time trying to reach the code in the subscriber. Is it really working?